### PR TITLE
Also treat NR code 0x7f as serviceNotSupported

### PR DESCRIPTION
**Pull request by akorb**

### CODE REVIEW

**Comment by polybassa** [comment]
Hi, there is a conflict
also did you check if uds_utils.uts still works?

**Comment by akorb** [comment]
Resolved and tests succeed on my machine for python2 and python3 with sudo

I only got 
```
WARNING: UNFINISHED THREADS
[<_MainThread(MainThread, started 139742538897216)>, <UDS_TesterPresentSender(Thread-158, started 139742230328896)>]
```
but that's probably unrealted to my changes.

File: scapy/contrib/automotive/enumerator.py [hunk 1]
```diff
@@ -611,7 +611,7 @@ def _evaluate_response(self, response, **kwargs):
         retry_if_busy_returncode = kwargs.pop("retry_if_busy_returncode", True)
 
         if exit_if_service_not_supported and response.service == 0x7f and \
-                self._get_negative_response_code(response) == 0x11:
+                self._get_negative_response_code(response) in [0x11, 0x7f]:
             log_interactive.debug("[-] Exit execute because negative response "
```
Comment (polybassa) [inline scapy/contrib/automotive/enumerator.py]: could you also add a note in this logging.. or maybe split this line into two evaluation to be able to provide a meaningful log message

Comment (akorb) [reply scapy/contrib/automotive/enumerator.py]: Done

**Review by polybassa — APPROVED** [review]

**Pull request merged**